Bottom: 0dc2984d849873929062e47b28101a4611901332
Top:    d5590ab2d9ddb8caa5ddb2372f732435ffddd7ca
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2014-01-27 12:11:55 -0800

Refresh of v18

---

diff --git a/configure.ac b/configure.ac
index 59c5eb1..7581df4 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1,7 +1,7 @@
 dnl Process this file with autoconf to produce a configure script.
 
 AC_PREREQ([2.63])
-AC_INIT([librdmacm],[1.0.17],[linux-rdma@vger.kernel.org])
+AC_INIT([librdmacm],[1.0.18],[linux-rdma@vger.kernel.org])
 AC_CONFIG_SRCDIR([src/cma.c])
 AC_CONFIG_AUX_DIR(config)
 AC_CONFIG_MACRO_DIR(config)
diff --git a/librdmacm.spec.in b/librdmacm.spec.in
index 7ad6c0c..15e0e31 100644
--- a/librdmacm.spec.in
+++ b/librdmacm.spec.in
@@ -1,7 +1,7 @@
 %define ver @VERSION@
 
 Name: librdmacm
-Version: 1.0.17
+Version: 1.0.18
 Release: 1%{?dist}
 Summary: Userspace RDMA Connection Manager
